### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AhsenKh**
Kindly review changes, and I am pretty sure the changes are a bit vague. So I am ready to clarify.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ The process by which UnCSS removes the unused rules is as follows:
 
 - UnCSS cannot be run on non-HTML pages, such as templates or PHP files. If you need to run UnCSS against your templates, you should probably generate example HTML pages from your templates, and run uncss on those generated files; or run a live local dev server, and point uncss at that.
 - UnCSS only runs the Javascript that is run on page load. It does not (and cannot) handle Javascript that runs on user interactions like button clicks. You must use the `ignore` option to preserve classes that are added by Javascript on user interaction.
+- UnCSS does not cater for classes used inside `ng-class` directive, and those classes are removed in the output.
```
Comment (RyanZim) [inline README.md]: Should be reworded, though this should be self-evident IMO.

File: README.md [hunk 2]
```diff
@@ -23,6 +23,8 @@ The process by which UnCSS removes the unused rules is as follows:
...
+- UnCSS also removes classes applied on child element, which are generated during rendering, even though they are being used.
```
Comment (RyanZim) [inline README.md]: Not sure what you're trying to say here?

Comment (AhsenKh) [reply README.md]:
```
.someClass label {
    style: something;
}
```

But in html, I have have a label element after `someClass` class. I create a label element on through Javascript. So, UnCSS would simply remove this css class.
It is understandable if I give an html file to clean. But for me, it even removed this class when I gave it the web page.

**Comment by RyanZim** [comment]
@AhsenKh ping?

File: README.md (hunk 2 above)
Comment (AhsenKh) [reply README.md]: Yes I do understand. However, it removed the rule for me even though it was present inside the div. I gave 10 seconds timeout, but didn't work. However, I shall make a simpler example and try to experiment again.

This is the class I believe was removed
```
md-input-container.md-input-invalid label {
    color: #D84315;
}
```

**Comment by RyanZim** [comment]
Closing for lack of response.

**Pull request closed without merging**